nwb_conversion_tools/utils/nwbephyswriterdatachunkiterator.py

```diff
@@ -18,12 +30,50 @@ def __init__(
         chunk_mb: float = None,
         chunk_shape: tuple = None,
     ):
+        """
+        Initialize an Iterable object which returns DataChunks with data and their selections on each iteration.
+
+        Parameters
+        ----------
+        ephys_writer : si.BaseRecording
+          The BaseRecording object (from spikeinterface/spikeinterface) which handles the API for data transfer.
+        segment_index : int, optional
+          DESCRIPTION. The default is 0.
```

Comment:
@Saksham20 or @alejoe91 Can one of you add the description for this? I have no idea what it does.